presto-main/src/main/java/com/facebook/presto/testing/RunLengthEncodedBlock.java

```diff
@@ -150,6 +156,12 @@ public void writeBytesTo(int position, int offset, int length, BlockBuilder bloc
     }
 
     @Override
+    public void writePositionTo(int position, BlockBuilder blockBuilder)
+    {
+        value.writeBytesTo(position, 0, getLength(position), blockBuilder);
```

Comment:
Shouldn't this call `value.writePositionTo`? Otherwise it's not going to work for Blocks that contain Objects